packages/local-signing-manager/src/types/index.ts

```diff
@@ -1,16 +1,9 @@
 /**
  * URI|mnemonic|hex representation of a private key
  */
-export type PrivateKey =
-  | {
-      uri: string;
-    }
-  | {
-      mnemonic: string;
-    }
-  | {
-      seed: string;
-    };
+export type PrivateKey = ({ uri: string } | { mnemonic: string } | { seed: string }) & {
+  derivationPath?: string;
```

Comment:
This type of comment should show up in auto complete suggestions. I find the extra context helpful when writing code.

```suggestion
  /**
   * (optional) a specify a derivation path to apply to the secret. This allows for multiple accounts to be created with only one secret
   *
   * @note if using this option the account can only be recreated by knowing the path and the secret. Be sure to use a consistent scheme when making the path
   */
  derivationPath?: string;
```

Feel free to change or improve the wording.

Reply:
Thanks, sure.